src/Main/Foedag.cpp

```diff
@@ -383,7 +383,7 @@ bool Foedag::initBatch() {
   m_projectFileLoader->registerComponent(
       new ProjectManagerComponent{m_compiler->ProjManager()},
       ComponentId::ProjectManager);
-  auto taskM = new TaskManager;
+  auto taskM = new TaskManager(m_compiler);
```

Comment:
Why do we need this change ?
I suspect we need to change upstream codebase too.

Reply:
compiler pointer pass to report manager. Report manager requires compiler pointer to recognize opensta.